crypto/sha/asm/sha1-x86_64.pl

```diff
@@ -945,18 +945,18 @@ ()
 	movaps	-40-1*16($fp),%xmm11
```

Comment:
I expect this to trap, because of unaliged access.

Reply:
Unaligned access on x86_64 should not trap, should it? Nevertheless this is wrong anyway.

Reply:
In general unaligned access does not trap, but this instruction is an exception to the general rule.
It is of course possible that an interrupt handler emulates this instruction, but I have little experience with
native windows, on linux it does simply trap.
Overall this PR is as wrong as it can possibly be, one additional bug is the stack unwinding code for these,
functions was not adjusted, and does still assume R11 holds the frame pointer:
https://github.com/openssl/openssl/blob/38e8392ba0c8dcd975de47a3119d0051cf5e44a1/crypto/sha/asm/sha1-x86_64.pl#L1972-L1980